test/appium/tests/test_chat_management.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+import time
+
+from tests import transaction_users
+from tests.base_test_case import MultipleDeviceTestCase
+from views.sign_in_view import SignInView
+
+
+@pytest.mark.all
+@pytest.mark.chat_management
+class TestChatManagement(MultipleDeviceTestCase):
+
+    @pytest.mark.testrail_case_id(3412)
+    def test_delete_1_1_chat(self):
+        self.create_drivers(2)
+        device_1 = self.drivers[0]
+        device_2 = self.drivers[1]
+        user_1 = transaction_users['A_USER']
+        user_2 = transaction_users['B_USER']
```

Comment:
transaction A/B users are used in tests with single session (please see tests.test_transaction.TestTransaction)
They could receive messages/requests on another device since tests are running in parallel, if they will be reused here.